.github/workflows/manubot.yaml

```diff
@@ -0,0 +1,40 @@
+name: Manubot
```

Comment:
Maybe you want to add a status badge to the readme:
`![](https://github.com/manubot/rootstock/workflows/manubot/badge.svg)`

Reply:
Oh wait, that wouldn't be for the specific manuscript repo. Is there a way to get that dynamically? Would a relative image link work, like `![](/workflows/manubot/badge.svg)`

Reply:
Oh wait, that wont refer to the specific manuscript. Would a relative image link work? `![](workflows/manubot/badge.svg)`

Reply:
The relative image link won't work, first because it's an SVG and second because that is not a file path relative to the repo content.

We could use the [absolute URL](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository). It would get rebranded to the manubot instance from `manubot/rootstock` during setup. Do you think badges are important given that you can now see the check or fail mark from the main page? I guess it makes it more clear for those unfamiliar with CI / GitHub actions?

Reply:
I do think that badges are more obvious to a larger audience that may not know to look for the pass/fail icons.